### PR TITLE
Use a js file for config

### PR DESCRIPTION
**Pull request by intelliot**
Allows for comments and more flexibility

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

Before: Config file was json

After: Config file is JavaScript

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ https
 *.js.map
 dist
 .secret_config.json
```
Comment (keefertaylor) [inline .gitignore:6]: can we get rid of this now? Should we delete `.secret_config.json`?

Comment (Stormtv) [reply .gitignore:6]: I think keeping it makes sense as anyone who already has the repo pulled down would end up committing the .secret_config.json on their next push

**Comment by intelliot** [comment]
@keefertaylor I'd like to keep the old config filename in .gitignore in case anyone still has the file hanging around from before.

> Can we get rid of the older files? If yes, can we remove them in this PR?

Is this just referring to `.secret_config.json` or are there other files?

> If no, then we should probably update the documentation to note there are multiple methods?

I don't understand this question.. what methods?

**Comment by intelliot** [comment]
Merging this for now but @keefertaylor please open an issue if there are other things to address

**Pull request merged**